docs/source/quickstart.rst

```diff
@@ -55,15 +55,29 @@ To build a model server docker image, you will also need to install
 `here <https://docs.docker.com/install/>`_.
 
 
-Hello World
------------
+Preface
+-------
 
-Before starting, let's prepare a trained model for serving with BentoML. Train a
+Before started, let's discuss how BentoML's project structure would look like. For most use-cases, users can follow this minimal scaffold
```

Comment:
We will provide this here without adding another `advanced guide` to project structure. I think we need to discuss more in terms of what we want to put on that advanced guide